src/v2/view-builder/utils/ChallengeViewUtil.js

```diff
@@ -83,5 +83,24 @@ export function doChallenge(view) {
         Util.redirect(deviceChallenge.href);
       }
     }));
+    break;
+  case Enums.APP_LINK_CHALLENGE:
+    view.title = loc('appLink.title', 'login');
+    view.add(View.extend({
+      className: 'app-link-content',
+      template: hbs`
+            {{{i18n code="appLink.content" bundle="login"}}}
```

Comment:
Use `{{` to escape the content.

Reply:
+1. Using triple curly braces prevents Handlebars from auto-escaping. Please use double bracket notation here:

```suggestion
            {{i18n code="appLink.content" bundle="login"}}
```